oioioi/contests/migrations/0021_rename_contests_contestview_user_id_timestamp_a0151775_idx_contests_co_user_id_9b1d73_idx_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.20 on 2025-04-15 22:08
```

Comment:
@MasloMaslane This is the biggest concern in the migration to Django 5.2.
Due to change of index handling we need to fix the names of the indexes
(see https://adamj.eu/tech/2020/07/27/how-to-modernize-your-django-index-definitions/)

Default migration method requires long downtime due to rebuilding of the indexes, this suppose to be zero-downtime method, but we need to make sure tha the "old_names" are correct.